aireview/src/aireview/infrastructure/github_client.py

```diff
@@ -56,27 +58,33 @@ async def submit_review(
     ) -> None:
         response: Response
         try:
+            logger.info(f"Submit review to pull request #{pr_number}")
             for comment in review.comments:
-                review_data = {
-                    "body": f"**{comment.type}**: {comment.content}",
-                    "commit_id": '',
-                    "path": comment.file_path,
-                    "start_line": comment.line,
-                    "start_side": 'RIGHT',
-                    "line": comment.line+1,
-                    "side": 'RIGHT'
-                }
-
-                response = self._session.post(f"{self._base_url}/repos/{self.owner}/{self.repo}/pulls/{pr_number}/comments",headers=self._headers, json=review_data)
-                response.raise_for_status()
+                logger.info(f"comment: {comment}")
```

Comment:
**style**: Empty commit_id in submit_review is a potential issue. This should be populated with the HEAD commit of the PR to ensure comments are attached to the correct version.